sw/airborne/modules/computer_vision/lib/vision/image.c

```diff
@@ -725,15 +824,25 @@ void image_draw_line_color(struct image_t *img, struct point_t *from, struct poi
   /* draw the line */
   for (uint16_t t = 0; /* starty >= 0 && */ starty < img->h && /* startx >= 0 && */ startx < img->w
        && t <= distance + 1; t++) {
+
+      // depending on startx being odd or even, we first have to set U or V
+      if(startx % 2 == 1) {
+          temp_color[0] = color[2];
+          temp_color[2] = color[0];
+      }
+      else {
+          temp_color[0] = color[0];
+          temp_color[2] = color[2];
+      }
```

Comment:
Don't forget to run fix code style. You can also change the settings of your editor so you don't have this problem in the future.

Reply:
Ran fix code style